include/caf/caf_overview.rst

```diff
@@ -0,0 +1,182 @@
+.. _caf_overview:
+
+Common Application Framework overview
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Common Application Framework (CAF) is a set of ready-for-use modules and events built on top of :ref:`event_manager`.
+Using CAF allows you to have a consistent event-based architecture in your application.
+
+One of the applications that are using the CAF modules in |NCS| is :ref:`nrf_desktop`.
+
+In an event-based application, parts of the application functionality are separated into isolated modules that communicate with each other using Event Manager's events.
```

Comment:
Those are not Event Manager's events. CAF and application defines events. CAF defines some predefined events that are used by the modules with dedicated functionality. Eg. buttons module is responsible for submitting button events (defined in CAF itself).

Reply:
I worked on this with @MarekPieta , so let's discuss this on chat before I make any edits.

Reply:
Discussed via chat. No changes needed.